R/RunStep.R

```diff
@@ -128,5 +128,27 @@ RunStep <- function(lStep, lData, lMeta, lSpec = NULL) {
     cli_detail = "h3"
   )
 
-  return(do.call(GetStrFunctionIfNamespaced(lStep$name), params))
+  if (lStep$name %in% c("dplyr::mutate", "mutate")) {
```

Comment:
@jonthegeek @lauramaxwell @samussiah did we have a prior way of using `mutate()` as a step in the `RunWorkflow()` process?

Reply:
Got it, I'll go ahead and close this PR then, if it ever needs to be revisited at least we know this achievable too

Reply:
This approach is pretty slick, even if it's perhaps adding more complexity than we want in `RunStep`.